smriprep/workflows/surfaces.py

```diff
@@ -1049,23 +1050,10 @@ def init_anat_ribbon_wf(name="anat_ribbon_wf"):
     workflow = pe.Workflow(name=name)
 
     inputnode = pe.Node(
-        niu.IdentityInterface(
-            fields=[
-                "white",
-                "pial",
-                "t1w_mask",
-            ]
-        ),
+        niu.IdentityInterface(fields=["white", "pial", "ref_file"]),
```

Comment:
`ref_file` is vague (any anatomical? masked?) - can you add a small description in the docstring?

Reply:
It's literally just a reference, but sure.